content/en/docs/ipv6.md

```diff
@@ -0,0 +1,63 @@
+---
+title: IPv6 Support
+slug: ipv6-support
+top_graphic: 1
+date: 2020-02-07
+lastmod: 2020-02-07
+---
+
+{{< lastmod >}}
+
+Let's Encrypt supports IPv6 both for accessing the ACME API using an ACME
+client, and for outbound domain validation requests made by our validation
+authority.
+
+## Domain Validation
+
+When making outbound domain validation requests for a domain that has both IPv4
+and IPv6 addresses (e.g. both `A` and `AAAA` records) Let's Encrypt will always
+prefer the IPv6 addresses for the initial connection. If the IPv6 connection
+fails at the network level (e.g. there is a timeout) and there are IPv4
+addresses available then we will retry the request with one of the IPv4
+addresses.
+
+## Incorrect IPv6 Addresses
+
+Often domain owners will not be aware of an `AAAA` record for their domain. If
+the IPv6 address in the `AAAA` record is incorrect it will affect the domain
+validation process.
+
+Commonly the IPv6 address will be a different server than the IPv4 address where
+the ACME client is being run. Since the ACME client only configures the IPv4
+server to respond to the challenge domain validation will fail when the IPv6
+server is used.
+
+In most cases the correct fix is to update the IPv6 address to point to the
+server the ACME client is running on, or to remove the `AAAA` record if the
+domain is not intended to work with IPv6. There is not a way to request Let's
+Encrypt prefer IPv4, you must fix the misconfiguration.
+
+## IPv6 to IPv4 Retry Details
+
+The IPv6 to IPv4 retry does **not** occur if there is an non-network level error.
```

Comment:
There are two negative in this sentence, so it would be clearer to remove them both. Also I think we only do this on connect timeouts.

```suggestion
The IPv6 to IPv4 retry only occurs on connection timeouts, not on other types of error.
```